app/src/main/java/com/github/polypoly/app/WelcomeActivity.kt

```diff
@@ -43,6 +79,18 @@ class WelcomeActivity : ComponentActivity() {
     fun WelcomeContent() {
         GameMusic.setSong(LocalContext.current, R.raw.mocksong)
         GameMusic.startSong()
+
+        val isSignedIn = remember { mutableStateOf(false) }
+        mAuthListener = FirebaseAuth.AuthStateListener {
+            val user = FirebaseAuth.getInstance().currentUser
+            isSignedIn.value = user != null
+            if(isSignedIn.value) {
+                print("\n__________________ Signed in: ${user?.displayName} _________________\n")
```

Comment:
Are these prints used for debug? Maybe you should remove them (when the PR is done)

Reply:
yes they are, I will remove them at the very end